app/dashboards/member_dashboard.rb

```diff
@@ -0,0 +1,62 @@
+require "administrate/base_dashboard"
+
+class MemberDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    events: Field::HasMany,
+    organizations: Field::HasMany,
```

Comment:
After playing around to understand administrate and that bug: https://github.com/montrealrb/Montreal.rb/pull/74/files#r47185636

I found out that removing all references in the dashboard to events and organizations for now fixes it.